packages/jest-circus/src/utils.ts

```diff
@@ -282,59 +284,61 @@ export const makeRunResult = (
   unhandledErrors: unhandledErrors.map(_formatError),
 });
 
+export const makeSingleTestResult = (
+  test: Circus.TestEntry,
+): Circus.TestResult => {
+  const {includeTestLocationInResult} = getState();
+  const testPath = [];
+  let parent: Circus.TestEntry | Circus.DescribeBlock | undefined = test;
+  do {
+    testPath.unshift(parent.name);
+  } while ((parent = parent.parent));
+
+  const {status} = test;
+
+  if (!status) {
+    throw new Error('Status should be present after tests are run.');
+  }
```

Comment:
```suggestion
  invariant(status, 'Status should be present after tests are run.');
```

also, check this before the `do-while`, just in case